src/libs/ReportUtils.js

```diff
@@ -1013,9 +1013,9 @@ function buildOptimisticWorkspaceChats(policyID, policyName) {
  * @returns {Boolean}
  */
 function isUnread(report) {
-    const lastReadSequenceNumber = report.lastReadSequenceNumber || 0;
-    const maxSequenceNumber = report.maxSequenceNumber || 0;
-    return lastReadSequenceNumber < maxSequenceNumber;
+    const lastMessageTimestamp = report.lastMessageTimestamp || 0;
+    const lastReadTimestamp = report.lastReadTimestamp || 0;
```

Comment:
In theory, `report.lastMessageTimestamp` should always be the last visible message.

Reply:
I'm not sure based on your response, but it sounds like maybe you're saying that if someone deletes a message that we will update a participant's report object with the `lastMessageTimestamp` of the last _visible_ message? Where does that happen exactly?

I appreciate that we are assuming this will work in theory, but we can add a test case for this in the QA steps and see whether we got it right.

Reply:
Ok, so based on my testing the lastMessageTime is always the last visible message.